src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -187,6 +198,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            string applicationId = GetApplicationId();
+
+            app.UseSwagger(o => o.RouteTemplate = applicationId + "/swagger/{documentName}/swagger.json");
+
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/" + applicationId + "/swagger/v1/swagger.json", "Altinn App API");
```

Comment:
This is highlighted as a code smell. Not sure how to fix it, but could you try using string interpolation?
```suggestion
                c.SwaggerEndpoint($"/{applicationId}/swagger/v1/swagger.json", "Altinn App API");
```

Reply:
Fixed